api/ontology/ontology.go

```diff
@@ -88,6 +88,26 @@ func ResourceTypes(r IsResource) []string {
 	return nil
 }
 
+// GetResourceTypes returns a list of resource types that are supported by the ontology.
+func GetResourceTypes() []string {
+	var types []string
+
+	var resource Resource
+
+	// Accessing the descriptor of the resource message.
+	md := resource.ProtoReflect().Descriptor()
+
+	// Durchlaufen der Felder der Resource
+	for i := 0; i < md.Fields().Len(); i++ {
+		field := md.Fields().Get(i)
+		if field.ContainingOneof() != nil {
```

Comment:
What is this about?

Reply:
Our owl2proto tool generates a oneOf field that currently includes every resource. If a bug in our implementation or a mistake in the ontology file leads to resources appearing outside the oneOf definition, those resources shouldn't displayed.

Reply:
I think you have to explain me it in person :D but for now I guess this is fine :)